#讯飞tts参数，这些参数除了text必填之外，都有默认值
#tts的种类，0：百度，1:科大讯飞
int32 tts_type

#合成音频数字发音方式
int32 	rdn		

#合成音频的音量
int32 	volume		

#合成音频的音调
int32 	pitch		

#合成音频对应的语速
int32 	speed		

#合成音频采样率
int32 	sample_rate	

#合成发音人
string 	voice_name	

#合成文本编码格式
string 	text_encoding	

#合成文本内容
string 	text
